src/dashboard/src/components/api/views.py

```diff
@@ -45,6 +45,39 @@
 UUID_REGEX = re.compile(r'^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$', re.IGNORECASE)
 
 
+def _api_endpoint(expected_methods, required_fields=[]):
```

Comment:
I don't think it's a problem in this case but lists as default values is generally a bad idea, right. Also, this `required_fields` argument is never used, so why have it?